mmv1/third_party/terraform/data_sources/data_source_compute_region_network_endpoint_group.go.erb

```diff
@@ -0,0 +1,77 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceGoogleComputeRegionNetworkEndpointGroup() *schema.Resource {
+	return &schema.Resource{
```

Comment:
I think this should follow the same formula as the non-region counterpart: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/data_sources/data_source_compute_network_endpoint_group.go#L12

As of right now, this data source only supports fields for `name`,`region`,`project`, and `self_link` and none of the other attributes from the resource version defined in the Attributes Reference